Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/RadialView.cs

```diff
@@ -196,6 +222,11 @@ public override void SolverUpdate()
                 goalRotation.x = goalRotation.z = 0f;
             }
 
+            if(UseFixedVerticalPosition == true)
```

Comment:
```suggestion
            if (UseFixedVerticalPosition)
```

Reply:
It's a boolean so we don't need to do the direct comparison to true (generally if params are named well like they are here, it can be fairly readable to be like "if using a fixed vertical position then..."

Reply:
Applied the update!